app/views/posts/index.html.erb

```diff
@@ -1,2 +1,34 @@
-<h1>Posts</h1>
-<p>This is to display all posts</p>
+<% #This page displays all the post of a user %>
+
+<section class='user-posts-container'>
+  <div class='user-posts-header'>
+    <%= image_tag @user.photo, alt: 'user photo', class: 'user-posts-img' %>
+    <div class='user-posts-text-container'>
+      <p class='users-posts-name'><%= @user.name %></p>
+      <p class='user-posts-number'>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+    </div>
+  
+  <ul>
+    <% @user.posts.each_with_index do |post, index| %>
+      <li class='user-posts-item-container'>
+        <div class='user-posts-item-content'>
+          Post <%= index + 1 %> <br><br>
+          <%= link_to post.title, user_post_path(@user, post) %>
+          <p><%= post.text %></p>
+          Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+        </div>
+        <ul class= 'user-posts-comment-container'>
+          <% post.recent_comments.each do |comment| %>
```

Comment:
- [OPTIONAL ]Great job working on the requirements so far but I think it would be great if you just add the likes and comment counters in the side as shown in the given design so please align them as the given design 🚀🔥💯💪🏻